gcore/src/general.rs

```diff
@@ -145,3 +145,31 @@ impl ActorId {
         &mut self.0[..]
     }
 }
+
+#[derive(Clone, Copy, Debug, Default, Hash, Ord, PartialEq, PartialOrd, Eq)]
+pub struct H256(pub [u8; 32]);
```

Comment:
This type's name is too similar to the `primitive_types::H256` and can be confused I guess.

Reply:
`Hash256` better?

Reply:
> `Hash256` better?

Maybe it should be more application-specific like `CodeHash`?

Reply:
> > `Hash256` better?
> 
> Maybe it should be more application-specific like `CodeHash`?

Agree.